core/restic/usr/local/bin/rclone-wrapper

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+# Rclone expects the password is in obscured format
+
+if [ -n "${RCLONE_SMB_PASS}" ]; then
+    RCLONE_SMB_PASS="$(/usr/bin/rclone obscure "${RCLONE_SMB_PASS}")"
+    export RCLONE_SMB_PASS
+fi
```

Comment:
I'd go with a more generic approach to be a little future proof: obscure all environment variables named `RCLONE_<something>_PASS`.

I'm not sure it's easy to do ...

Reply:
ok